service/src/test/java/com/derpgroup/derpwizard/dao/impl/DynamoEmbeddedUserPreferencesDAOTest.java

```diff
@@ -0,0 +1,31 @@
+package com.derpgroup.derpwizard.dao.impl;
+/*
```

Comment:
This is a problem for later - the shell of the embedded impl is here, it just needs to be built out once dependency issues are resolved.